app/src/main/java/com/babylon/wallet/android/presentation/settings/linkedconnectors/LinkedConnectorsScreen.kt

```diff
@@ -279,7 +274,16 @@ fun LinkedConnectorsContentWithoutActiveLinkedConnectorsPreview() {
 fun LinkedConnectorsContentWithActiveLinkedConnectorsPreview() {
     RadixWalletTheme {
         LinkedConnectorsContent(
-            activeLinkedConnectorsList = SampleDataProvider().p2pLinksSample.toPersistentList(),
+            activeLinkedConnectorsList = listOf(
+                LinkedConnectorsUiState.ConnectorUiItem(
+                    id = "public_key",
```

Comment:
In this case, the format of the IDs doesn't matter as they are not displayed. From the UI perspective it can be any String that's unique inside the list.